src/components/Todo/todo.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { addTodo, handleCreateBtn, handleEmptyError } from "actions";
+import Navbar from "components/Todo/Navbar/navbar";
+import TopBar from "components/Todo/Topbar/top-bar";
+import AddCard from "components/Todo/AddCard/add-card";
+import TodoViews from "components/Todo/TodoViews/todo-views";
+
+const Todo = () => {
+  const dispatch = useDispatch();
+
+  const [inputData, setInputData] = useState("");
+
+  const todoList = useSelector((state) => state.todoReducers.list);
+  const isCreateBtnClicked = useSelector(
+    (state) => state.handleTokens.isCreateBtnClicked
+  );
+  const isEmptyError = useSelector((state) => state.handleErrors.isEmptyError);
+
+  const toggleEmptyError = () => {
+    dispatch(handleEmptyError(isEmptyError));
+  };
+  const handleCreateClick = () => {
+    dispatch(handleCreateBtn(isCreateBtnClicked));
+  };
+  const sanitizeInput = (input) => {
+    return input.replace(/(<([^>]+)>)/g, "");
+  };
+  const handleInputChange = (e) => {
+    setInputData(e.target.value);
+    if (isEmptyError) toggleEmptyError();
+  };
+  const handleAddTask = (e) => {
+    const sanitizedData = sanitizeInput(inputData);
+    if (sanitizedData.trim()) {
+      dispatch(addTodo(sanitizedData));
+      setInputData("");
+      handleCreateClick();
+
+      if (isEmptyError) toggleEmptyError();
+    } else {
+      if (!isEmptyError) toggleEmptyError();
+    }
+  };
+  const handleKeyDown = (e) => {
+    if (e.key === "Enter") {
```

Comment:
Can we use a const here?
```suggestion
    if (e.key === ENTER) {
```

Reply:
vaia `e.key===ENTER` is raising error. e.key has the property `"Enter"`